libs/coin-modules/coin-aptos/src/__tests__/logic/combine.unit.test.ts

```diff
@@ -0,0 +1,78 @@
+import { createFixtureAccount } from "../../bridge/bridge.fixture";
+import { Hex, generateSignedTransaction } from "@aptos-labs/ts-sdk";
+import { combine } from "../../logic/combine";
+
+const hexRawTx =
+  "0xfdde1012c0fac1f9a121eb3c8481c90d473df1c4180c070bd4f2549a6d06180400000000000000000200000000000000000000000000000000000000000000000000000000000000010d6170746f735f6163636f756e74087472616e736665720002203f5f0fcc8a909f23806e5efbdc1757e653fcd744de516a7de12b99b8417925c1080a00000000000000400d03000000000064000000000000002c721b6800000000b7";
+
+jest.mock("@aptos-labs/ts-sdk", () => {
+  const originalModule = jest.requireActual("@aptos-labs/ts-sdk");
+  const partialMockedModule = Object.keys(originalModule).reduce(
+    (pre: { [key: string]: jest.Mock }, methodName) => {
+      pre[methodName] = jest.fn();
+      return pre;
+    },
+    {} as { [key: string]: jest.Mock },
+  );
+  return {
+    ...partialMockedModule,
+    // mock all except these
+    AccountAddress: originalModule.AccountAddress,
+    Deserializer: originalModule.Deserializer,
+    Hex: originalModule.Hex,
+    RawTransaction: originalModule.RawTransaction,
+    generateSignedTransaction: jest
+      .fn()
+      .mockImplementation(() => Hex.fromHexString(hexRawTx).toUint8Array()),
+    Serializer: originalModule.Serializer,
+    SignedTransaction: originalModule.SignedTransaction,
+  };
+});
```

Comment:
this is overkill, you don't need to do this, when you mock a module, you can just add the methods you're "going to use".

Reply:
I need the original implementation of the other classes/functions in some parts of the code, for that reason, I'm setting up the original version

Reply:
changed.